### PR TITLE
LPS-116225 create test for color list

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related issue: https://issues.liferay.com/browse/LPS-116225

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/color/Color.es.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+		const colorText = container.querySelector('.color-text');
+
+		expect(colorText.textContent).toBe('#2BA676');
+
+		const colorViewer = container.querySelector('.color-viewer');
+
+		expect(colorViewer).toBeDefined();
+
+		expect(asFragment()).toMatchSnapshot();
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/color/Color.es.js:34]: @marcosapmf, do we really need to generate a snapshot? Can you try to not use it as Alininha did [here](https://github.com/natocesarrego/liferay-portal/pull/641/commits/89ae97291405b1ecac75632f36e69985c284ac5c)?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/list/List.es.js [hunk 2]
```diff
@@ -36,6 +36,22 @@ describe('List', () => {
...
+			type: 'color',
+		};
+
+		const {asFragment, container} = render(<List {...colorProps} />);
+
+		const colorText = container.querySelectorAll('.color-text');
+
+		expect(colorText.length).toBe(5);
+
+		expect(asFragment()).toMatchSnapshot();
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/list/List.es.js:52]: @marcosapmf, please see my previous comment.

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**